subjects/devops/kept-promise/README.md

```diff
@@ -0,0 +1,52 @@
+## kept-promise
+
+### Instructions
+
+Thing in your code does not always happen as fast as you would like.
+
+Sometimes it is important to wait for a function to finish before keep going on.
+
+Create an asynchronous function `processInfo` in a file `kept-promise.js`. The function `processInfo` will take an asynchronous function as an input, and it will print the message `Ok!` if the return of the input function is zero or a multiple of two, `Error!` otherwise.
+
+The following function can be used as an example of input function for `processInfo`:
+
+```js
+const getImportantInfo = _ =>
+  new Promise(resolve => {
+    setTimeout(() => resolve(Math.round(Math.random() * 10)), 1000)
```

Comment:
This is out of the scope of the solution, which is why I haven't included anything about it since it is not "required knowledge". Adding something related to `setTimeout` might be confusing in my opinion

Reply:
We agreed to remove it since it's not needed.